### ISSUE TITLE
aiohttp and uvloop - crash

### ISSUE THREAD

**Issue by nibrag** [MagicStack/uvloop#34]
* **uvloop version**: 0.4.31
* **python version**: Python 3.5.2rc1
* **platform**: Linux debian-7 3.2.0-4-amd64

I'm not sure that this is a problem uvloop.
https://github.com/KeepSafe/aiohttp/issues/925

```
import asyncio
import aiohttp
import uvloop
from aiohttp.resolver import AsyncResolver

async def run(_loop):
    r = AsyncResolver(loop=_loop)
    conn = aiohttp.TCPConnector(loop=_loop, resolver=r)

    with aiohttp.ClientSession(connector=conn, loop=_loop) as ses:
        async with ses.get('http://vk.com') as resp:
            if resp.status == 200:
                print(await resp.read())
            else:
                print(resp.status)

if __name__ == '__main__':
    asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
    loop = asyncio.get_event_loop()
    loop.run_until_complete(run(loop))
    loop.close()
```
Result:
```
python3: src/unix/core.c:888: uv__io_stop: Assertion `loop->watchers[w->fd] == w' failed.
bash: line 1:  4020 aborted  /usr/bin/python3 -u /vagrant/test.py
python finished with exit code 134
```

**Comment by 1st1** [comment]
Thanks a lot for reporting this. I'll take a look on Monday.

**Comment by 1st1** [comment]
Should be fixed in `v0.4.32`. Please check.

**Comment by nibrag** [comment]
It works, thanks.
